oneapp-server/types/ApplicationContact.js

```diff
@@ -8,55 +8,55 @@ const typeDef = gql`
 
   type ApplicationContact {
     "Applicant first name"
-    APPLICANT_FIRST_NAME: String
+    APPLICANT_FIRST_NAME: String! @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Applicant last name"
-    APPLICANT_LAST_NAME: String
+    APPLICANT_LAST_NAME: String! @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Applicant middle name"
-    APPLICANT_MIDDLE_NAME: String
+    APPLICANT_MIDDLE_NAME: String @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Application maiden name"
-    APPLICANT_MAIDEN_NAME: String
+    APPLICANT_MAIDEN_NAME: String @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Payee first name, if different than applicant"
-    PAYEE_FIRST_NAME: String
+    PAYEE_FIRST_NAME: String! @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Payee last name, if different than applicant"
-    PAYEE_LAST_NAME: String
+    PAYEE_LAST_NAME: String! @constraint(pattern: "^[a-zA-Z.''-'\s]*$", maxLength: 25)
     "Is applicant homeless"
     IS_HOMELESS: Boolean
     "Applicant address line 1"
-    ADDRESS1: String
+    ADDRESS1: String @constraint(pattern: "^.*(?=.*[a-zA-Z])([a-zA-Z0-9'#@&/-_.,\s])*$", maxLength: 50)
     "Applicant address line 2"
-    ADDRESS2: String
+    ADDRESS2: String @constraint(pattern: "^[a-zA-Z0-9.,'#@&\-_/\s]*$", maxLength: 50)
     "Applicant address city"
-    CITY: String
+    CITY: String @constraint(pattern: "^[a-zA-Z0-9.,'#@&\-_/\s]*$", maxLength: 50)
     "Applicant address state"
     STATE: String
     "Applicant address zipcode"
-    ZIP: Int
+    ZIP: Int @constraint(maxLength: 5)
     "Applicant address zipcode extend 4"
     ZIP4: Int
     "New Jersey county id, retrieved from zipcode to county lookup"
-    COUNTY_NUMBER: Int
+    COUNTY_NUMBER: Int!
     "Mailing address line 1"
-    M_ADDRESS1: String
+    M_ADDRESS1: String @constraint(pattern: "^.*(?=.*[a-zA-Z])([a-zA-Z0-9'#@&/-_.,\s])*$", maxLength: 50)
     "Mailing address line 2"
-    M_ADDRESS2: String
+    M_ADDRESS2: String @constraint(pattern: "^[a-zA-Z0-9.,'#@&\-_/\s]*$", maxLength: 50)
     "Mailing address city"
-    M_CITY: String
+    M_CITY: String @constraint(pattern: "^[a-zA-Z0-9.,'#@&\-_/\s]*$", maxLength: 50)
     "Mailing address state"
     M_STATE: String
     "Mailing address zipcode"
-    M_ZIP: Int
+    M_ZIP: Int @constraint(maxLength: 5)
```

Comment:
Zip codes look to have some additional regex validation. Is the legacy ASPX using a single form field for zip and breaking into two separate db fields? If so, we should set up these fields to work with that validation, i.e. zip should also have minLength = 5 and zip4 should have max/min = 4.